lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -83,12 +104,28 @@ dampening_suppress_time:
 
 default_information:
   kind: boolean
-  config_get_token_append: '/^default-information originate$/'
-  config_set_append: '<state> default-information originate'
-  default_value: false
+  cli_nexus:
+    config_get_token_append: '/^default-information originate$/'
+    config_set_append: ' <state> default-information originate'
+    default_value: false
+  cli_ios_xr:
+    # IOS XR does not have default-information under address-family,
+    #   so we must replace config get/set
+    # Perhaps better to fix in ruby code?
```

Comment:
FYI:

It's not supported under bgp af:

RP/0/RP0/CPU0:ios(config)#router bgp 5
RP/0/RP0/CPU0:ios(config-bgp)#address-family ipv4 unicast
RP/0/RP0/CPU0:ios(config-bgp-af)#bgp d?
dampening
RP/0/RP0/CPU0:ios(config-bgp-af)#bgp ?
  attribute-download  Configure attribute download for this address-family
  bestpath            Change default route selection criteria
  client-to-client    Configure client to client route reflection
  dampening           Enable route-flap dampening
  label-delay         Specify delay for batching label processing
  origin-as           BGP origin-AS knobs
  scan-time           Configure background scanner interval for this address-family

So _exclude here for XR, then add the logic minitest to catch the UnsupportedError. Unless it's a special case where for example it works under the af for a default vrf, but not a non-default.